Jenkinsfile

```diff
@@ -113,14 +113,20 @@ void runTest(String testName) {
     """
 }
 
-String distSubDir() {
+String distSubDir(boolean tested) {
```

Comment:
I'm sure I'm getting confused but does Groovy allow skipping specifying parameters in a method call when the parameter is not marked as optional?

https://github.com/SUSE/scf/blob/3cb8b3600a4e4f82e6f410ac7ed9ad1cb8b0f27c/Jenkinsfile#L579

Reply:
Good catch, fixed in 964821c4c2c8c602300cdfe42247e7225ea25781.